### DISCUSSION TITLE
peer for parent or outer element

### DISCUSSION THREAD

**Discussion by sonmezcem** [tailwindlabs/tailwindcss#13545] [Help] [answered]
Hi,

I am trying to peer work on parent element but couldn't make it work.

Following one works as expected ;

https://play.tailwindcss.com/AEP69AFfTP

but this one does not work. 

https://play.tailwindcss.com/iIkv6Jq8wl

I tried to make it work for several hours and look for discussions but I couldn't find a way to do make it work. Any help appreciated.

**Comment by wongjn** [comment, marked as answer]
You could consider making the parent `<div>` the `peer` and then use `:has()` via `peer-has-*`: https://play.tailwindcss.com/jpQ8956Bsy

**Reply by sonmezcem** [reply]
Thank you for your help. I finally make it work with your help.
